lib/bootstrap.test.js

```diff
@@ -42,7 +42,7 @@ describe(_.startCase(filename), function () {
       subsite: 'example4'
     }];
 
-    bootstrapFake = files.getYaml(path.resolve('./test/fixtures/config/bootstrap'));
+    bootstrapFake = files.getYaml(path.resolve('./test/fixtures/config/bootstrap')); // eslint-disable-line
```

Comment:
what error was eslint throwing?

Reply:
It was saying it's an unused variable.
I didn't take the time to look to see if this is actually needed for tests or if a recent change removed the requirement. I'll give it another look and maybe just delete the line if possible.

Reply:
It wasn't needed, removed!